_scala3-reference/language-versions.md

```diff
@@ -1,36 +1,11 @@
 ---
 title: Language Versions
 type: chapter
-description: This page lists the different flavours of language supported by the Scala 3 compiler.
 num: 100
-previous-page: overview
+previous-page: /scala3/reference/syntax
+next-page: /scala3/reference/language-versions/source-compatibility
 ---
 
-The default Scala language version currently supported by the Dotty compiler is `3.0`. There are also other language versions that can be specified instead:
+This chapter explains how different versions of the language interact with each regarding their inputs and outputs.
```

Comment:
I am not an english native, but “how they interact with each” does not sound correct to me, could you please double-check?

Reply:
Good catch. This was supposed to be `with each other`